foodsaving/conversations/api.py

```diff
@@ -168,5 +168,6 @@ class RetrieveConversationMixin(object):
     def retrieve_conversation(self, request, *args, **kwargs):
         target = self.get_object()
         conversation = Conversation.objects.get_or_create_for_target(target)
-        serializer = ConversationSerializer(conversation, context={'request': request})
+        serializer = ConversationSerializer(conversation, data={}, context={'request': request})
```

Comment:
This permission check seems to be duplicated. If the user isn't allowed to access the parent endpoint (e.g. group detail, pickup), then the queryset gets filtered and they also can't access the `/pickup/<id>/conversation` endpoint.

But maybe that's because the membership management is also duplicated in the conversation?

Reply:
The conversation is only for people who are signed up to the pickup, so it's legitimate to have access to the pickup, but not the conversation.